x-pack/plugins/aiops/public/components/change_point_detection/change_point_detection_context.tsx

```diff
@@ -163,7 +163,7 @@ export const ChangePointDetectionContextProvider: FC = ({ children }) => {
       ({ aggregatable, esTypes, displayName }) =>
         aggregatable &&
         esTypes &&
-        esTypes.includes('keyword') &&
+        esTypes.some((el) => ['keyword', 'ip'].includes(el)) &&
```

Comment:
we could use `ES_FIELD_TYPES` for these types.

Reply:
yes, it's possible, probably the data view plugin should use it as well. At the moment they are defined as strings https://github.com/elastic/kibana/blob/9a5df11ea07d9512491b141368f7a31e7abaf2e7/src/plugins/data_views/common/types.ts#L396

Reply:
Updated in https://github.com/elastic/kibana/pull/151022/commits/c7fd09448811b13f3124bb511fc7c94e3486cf67